docs/datamap/lucene-datamap-guide.md

```diff
@@ -0,0 +1,133 @@
+# CarbonData Lucene DataMap (Alpha feature in 1.4.0)
+  
+* [DataMap Management](#datamap-management)
+* [Lucene Datamap](#lucene-datamap-introduction)
+* [Loading Data](#loading-data)
+* [Querying Data](#querying-data)
+* [Data Management](#data-management-with-pre-aggregate-tables)
```

Comment:
It's incorrect here:
`data-management-with-pre-aggregate-tables`
It should be
`data-management-with-lucene-datamap`